src/foam/dao/EasyDAO.js

```diff
@@ -228,6 +228,23 @@ foam.CLASS({
         if ( getLastModifiedByAware() )
           delegate = new foam.nanos.auth.LastModifiedByAwareDAO.Builder(getX()).setDelegate(delegate).build();
 
+        if ( getApprovableAware() && getApprovableAwareEnabled() )
```

Comment:
The logic below, if getApprovableAwareEnabled is false because interface decorators are false then getApprovableAware will also be false.  So the second setup below doesn't make sense.  
I added the comment below before this one.

Reply:
Done.